doc/tech/approach.md

```diff
@@ -129,7 +129,7 @@ To allow multiple separated simultaneous SW Virtualization domains onto a shared
 on suitable protocols e.g. VxLAN with their VNI Ranges. This separation must be done through an
 administrative domain that could not be compromised by any of the individual SW Virtualization domains either by malicious or unintentional Underlay Network mapping or configuration.
 
-The ETSI NFVI Infrastructure Management thus must be split into two distinct layers that can be referred to as HW Infrastructure Layer and SW Infrastructure Layer which can be managed separately from different administrative domains. When there are multiple separated simultaneous SW Virtualization domains, they have to be possible to be individual administrative domains.
+The ETSI NFV Infrastructure Management (that CNTT refer to as Cloud Infrastructure Management) thus must be split into two distinct layers that can be referred to as HW Infrastructure Layer and SW Infrastructure Layer which can be managed separately from different administrative domains. When there are multiple separated simultaneous SW Virtualization domains, they have to be possible to be individual administrative domains.
```

Comment:
@TFredberg I thing you are trying to split, "The ETSI NFV Infrastructure (that CNTT refers to as the Cloud Infrastructure)" into "into two distinct layers that can be referred to as HW Infrastructure Layer and SW Infrastructure Layer"  and not "split" the "The ETSI NFV Infrastructure Management". May I suggest, 

"The ETSI NFV Infrastructure (that CNTT refers to as the Cloud Infrastructure) can be considered to be composed of two distinct layers, referred to as HW Infrastructure Layer and SW Infrastructure Layer, which can then be managed separately from different administrative domains."

Reply:
Fixed